velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -35,9 +35,19 @@ enum class Table : uint8_t;
 
 namespace facebook::velox::exec::test {
 
+struct AggregationConfig {
+  std::vector<std::string> groupingKeys;
+  std::vector<std::string> aggregates;
+};
+
 struct PushdownConfig {
   common::SubfieldFilters subfieldFiltersMap;
   std::string remainingFilter;
+  // Aggregation pushdown configuration
+  std::optional<AggregationConfig> aggregationConfig;
+  // If true, insert an identity Project node before aggregation to block
+  // pushdown
+  bool insertProjectToBlockPushdown = false;
```

Comment:
We don't need this to be part of the config, just pass it as a separate parameter to `makeScanTask`